ee/clickhouse/queries/breakdown_props.py

```diff
@@ -0,0 +1,100 @@
+from typing import Dict, List
+
+from ee.clickhouse.client import sync_execute
+from ee.clickhouse.models.property import parse_prop_clauses
+from ee.clickhouse.queries.trends.util import populate_entity_params
+from ee.clickhouse.queries.util import parse_timestamps
+from ee.clickhouse.sql.person import GET_LATEST_PERSON_DISTINCT_ID_SQL, GET_LATEST_PERSON_SQL
+from ee.clickhouse.sql.trends.top_elements import TOP_ELEMENTS_ARRAY_OF_KEY_SQL
+from ee.clickhouse.sql.trends.top_person_props import TOP_PERSON_PROPS_ARRAY_OF_KEY_SQL
+from posthog.models.entity import Entity
+from posthog.models.filters.filter import Filter
+
+
+class BreakdownProps:
```

Comment:
judging by how this class is used, it makes more sense for these to be pure functions instead?

i.e. `get_person_prop_values` and `get_event_prop_values` both just need the same params as those passed to `__init__` right now, and can deterministically figure out the prop values without depending on any extra external state.